File: Selenium/Activities/HelloSelenium.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class HelloSelenium {

    public static void main(String[] args) {

        // setup the firefox driver (Geckodriver)
        WebDriverManager.firefoxdriver().setup();

        // Create a new instance of the firefox driver
        WebDriver driver = new FirefoxDriver();

        //Open the browser to a webpage
        driver.get("https://training-support.net");

        // Print the title of the page
        System.out.println("Title of the page is: " +driver.getTitle());


        driver.quit();
    }

}

File: Python/Activities/hello_selenium.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.common.by import By


service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver: 

    # Open the browser to a URL
    driver.get("https://training-support.net")
    
    #Print the title of the page
    print(f"The title of the page is: {driver.title}")
    print(f"Current url of the page is: {driver.current_url}")

    driver.find_element(By.LINK_TEXT, "About Us").click()



    #Close the browser
    driver.quit();
